index.html

```diff
@@ -1,42 +1,314 @@
+              <h2 class="grid-head fonts-1 p-color fw-n">Profesional Art</h2>
+              <h2 class="grid-head fonts-1 p-color fw-n">Printing Data</h2>
+              <p class=" grid-text fonts-4 p-color">A daily selection of privately
+                personalized reads; no accounts or
+                sign-ups required. has been the
+                industry's standard</p>
+              <ul class="tags">
+                <li class="tag tag-color fonts-4">html</li>
+                <li class="tag tag-color fonts-4">boostrap</li>
+                <li class="tag tag-color fonts-4">Ruby</li>
+              </ul>
+            </div>
+            <button class="button fonts-4 button-container" type="button">See Project</button>
+          </div>
+
+          <div class="card-holder">
+            <div class="card-works">
+              <h2 class="grid-head fonts-1 p-color fw-n">Profesional Art</h2>
+              <h2 class="grid-head fonts-1 p-color fw-n">Printing Data</h2>
+              <p class=" grid-text fonts-4 p-color">A daily selection of privately
+                personalized reads; no accounts or
+                sign-ups required. has been the
+                industry's standard</p>
+              <ul class="tags">
+                <li class="tag tag-color fonts-4">html</li>
+                <li class="tag tag-color fonts-4">boostrap</li>
+                <li class="tag tag-color fonts-4">Ruby</li>
+              </ul>
+            </div>
+            <button class="button fonts-4 button-container" type="button">See Project</button>
+          </div>
+
+          <div class="card-holder">
+            <div class="card-works">
+              <h2 class="grid-head fonts-1 p-color fw-n">Profesional Art</h2>
+              <h2 class="grid-head fonts-1 p-color fw-n">Printing Data</h2>
+              <p class=" grid-text fonts-4 p-color">A daily selection of privately
+                personalized reads; no accounts or
+                sign-ups required. has been the
+                industry's standard</p>
+              <ul class="tags">
+                <li class="tag tag-color fonts-4">html</li>
+                <li class="tag tag-color fonts-4">boostrap</li>
+                <li class="tag tag-color fonts-4">Ruby</li>
+              </ul>
+            </div>
+            <button class="button fonts-4 button-container" type="button">See Project</button>
+          </div>
+        </section>
+      </section>
+    </section>
+
+    <section class="about headline">
+      <div class="about-content">
+        <div class="padding-lf">
+          <h2 class="about-head primary-text-h1 fonts-1">About me</h2>
+          <p class="about-text content p-color fonts-4">Hello I’m a software developer! I can help
+            you build a product , feature or website
+            Look through some of my work and
+            experience! If you like what you see and
+            have a project you need coded, don’t
```

Comment:
![image](https://user-images.githubusercontent.com/43698511/130326194-a60861f6-ba78-48da-94b9-68f024c309ee.png)

- [ ] Kindly add the missing line in the about me section.
- [ ] The titles language, frameworks and skills should be aligned.
- [ ] Kindly add some margins. Check [the original design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=48%3A27) for refrence.